stanford_courses.info

```diff
@@ -7,12 +7,12 @@ project = stanford_courses
 dependencies[] = ctools
 dependencies[] = date
 dependencies[] = date_popup
-dependencies[] = features (>=2.0)
-dependencies[] = feeds (>=2.0-alpha8)
+dependencies[] = features
```

Comment:
Please restore the version numbers.
Features when exporting does not maintain these. 
Pain in da butt.